drivers/audio/wm8904.c

```diff
@@ -1210,6 +1214,30 @@ static int wm8904_configure(FAR struct audio_lowerhalf_s *dev,
           break;
 #endif /* CONFIG_AUDIO_EXCLUDE_VOLUME */
 
+#ifndef CONFIG_AUDIO_EXCLUDE_BALANCE
+        case AUDIO_FU_BALANCE:
+          {
+            /* Set the balance.  The percentage level * 10 (0-1000) is in the
+             * ac_controls.b[0] parameter.
+             */
+
+            uint16_t balance = caps->ac_controls.hw[0];
+            audinfo("    Balance: %d\n", balance);
+
+            if (balance >= 0 && balance <= 1000)
```

Comment:
Why not to change the application side to use 0-65535 range as well?

Reply:
Thanks for reviewing! From "apps/system/nxplayer/nxplayer.c" (line 2142) and "apps/system/nxplayer/nxplayer_main.c" (line 120) I deduced that applications seem to assume a range from 0 to 1000 for volume and balance.

Reply:
Ok, so let keep it this way :-)